src/scss/components/checkbox/_index.scss

```diff
@@ -142,120 +120,73 @@ hx-checkbox-control {
 
     &:checked,
     &:indeterminate {
-      color: @cyan-900;
+      color: $cyan-900;
 
       ~ label > hx-checkbox {
-        border-color: @cyan-900;
-        color: @cyan-900;
+        border-color: $cyan-900;
+        color: $cyan-900;
       }
     }
 
     &:valid {
       ~ label > hx-checkbox:hover {
-        background-color: @cyan-50;
-        border-color: @cyan-500;
-        color: @cyan-500;
+        @include hxCheckboxFacade($pseudo-state: hover);
       }
     }
   }
 }
 
-/* ========== INVALID ========== */
-.isInvalid() {
-  color: @red-900;
-
-  &::-ms-check {
-    border-width: 2px;
-    border-color: @red-900 !important;
-    color: @red-900 !important;
-  }
+// ========== INVALID ==========
+hx-checkbox-control[hx-dirty] > input[type="checkbox"]:invalid,
+hx-checkbox-control.hxInvalid > input[type="checkbox"]:enabled {
+  @include hxCheckboxControl;
```

Comment:
This should probably be...

```scss
@include hxCheckboxControl(invalid);
```

Reply:
Good catch.  Added the `invalid` state to the `hxCheckboxControl` mixin.